src/sql/mod.rs

```diff
@@ -720,7 +720,7 @@ mod tests {
             // Should have two nodes: source and "users" base table
             let ncount = mig.graph().node_count();
             assert_eq!(ncount, 2);
-            assert_eq!(get_node(&inc, &*mig, "users").name(), "users");
```

Comment:
Does the `&*` make a difference in these tests? They seem to be passing fine without it as well, so I removed it - let me know if it should be there though and I'll revert the commit.

Reply:
I think this is probably a leftover from when migrations worked somewhat differently, and can indeed be removed.